src/coreclr/jit/emitloongarch64.cpp

```diff
@@ -5048,10 +5048,14 @@ regNumber emitter::emitInsTernary(instruction ins, emitAttr attr, GenTree* dst,
 
         // TODO-LOONGARCH64-CQ: here sign-extend dst when deal with 32bit data is too conservative.
         if (EA_SIZE(attr) == EA_4BYTE)
+        {
             emitIns_R_R_I(INS_slli_w, attr, dst->GetRegNum(), dst->GetRegNum(), 0);
+        }
     }
     else
     {
+        assert((dst->OperGet() == GT_ADD) || (dst->OperGet() == GT_SUB));
```

Comment:
```suggestion
        assert(dst->OperIs(GT_ADD, GT_SUB));
```

similarly elsewhere.

Reply:
Already updated here and elsewhere in this commit, thank you very much!